### PR TITLE
update kubernetes and helm exporter docs to include information about license acceptance

### PR DESCRIPTION
**Pull request by nellshamrell**
Signed-off-by: Nell Shamrell <nellshamrell@gmail.com>

### CODE REVIEW

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb [hunk 1]
```diff
@@ -140,6 +140,63 @@ The Kubernetes exporter is an additional command line subcommand to the standard
...
+        value: accept-no-persist
+    ```
+
+7. You can run this manifest in your Kubernetes cluster with:
+
+    ```shell
+    $ kubectl create -f my_app.yaml
+    ```
+
+8. This will create a Kubernetes Stateful set running your package. To access the pod running your package from the outside internet, you will need to add a Kubernetes service (i.e. a Kubernetes load balancer) with an external IP. Here is an example.
```
Comment (raskchanky) [inline www/source/partials/docs/_dev-pkgs-exports.html.md.erb]: ~Stateful set~ StatefulSet

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb [hunk 2]
```diff
@@ -140,6 +140,63 @@ The Kubernetes exporter is an additional command line subcommand to the standard
...
+        ## An object containing parameters that affects how the Habitat service
+        ## is executed.
+        service:
+        ## Name of the Habitat service.
+        name: sample-node-app
+        ## Habitat topology of the service.
+        topology: standalone
+        env:
+        - name: HAB_LICENSE
+        value: accept-no-persist
```
Comment (raskchanky) [inline www/source/partials/docs/_dev-pkgs-exports.html.md.erb]: Does the indentation level of `value` need to match that of `name`?  As it is now, it looks like `value` is a peer of `env`, which doesn't seem right.

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb (hunk 1 above)
Comment (nellshamrell) [reply www/source/partials/docs/_dev-pkgs-exports.html.md.erb]: Good catch! Fixing!

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb (hunk 2 above)
Comment (nellshamrell) [reply www/source/partials/docs/_dev-pkgs-exports.html.md.erb]: That was likely copy pasta - fixing!

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb (hunk 1 above)
Comment (nellshamrell) [reply www/source/partials/docs/_dev-pkgs-exports.html.md.erb]: Fixed!

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb (hunk 2 above)
Comment (nellshamrell) [reply www/source/partials/docs/_dev-pkgs-exports.html.md.erb]: Fixed!

**Comment by nellshamrell** [comment]
@habitat-sh/docs-team

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb [hunk 3]
```diff
@@ -140,6 +140,63 @@ The Kubernetes exporter is an additional command line subcommand to the standard
     ```shell
     $ hab pkg export kubernetes --push-image --username <your_docker_hub_username> --password <your_docker_hub_password> -o my_app.yaml
     ```
+6. Please note that you will need to add the HAB_LICENSE environment variable to the generated manifest yaml file. One way to do this is to add the environmental variable directly to the generated manifest like this example:
```
Comment (kagarmoe) [inline www/source/partials/docs/_dev-pkgs-exports.html.md.erb]:
```suggestion
6. Add the HAB_LICENSE environment variable to the generated manifest yaml file. For example, add the environmental variable directly to the generated manifest:
```

File: www/source/partials/docs/_dev-pkgs-exports.html.md.erb [hunk 4]
```diff
@@ -165,6 +222,8 @@ Additionally, the Kubernetes Habitat operator is automatically added to the Helm
 
 4. More information on how to setup Helm and use of the Helm exporter can be found on the [announcement blog](https://www.habitat.sh/blog/2018/02/Habitat-Helm/)
 
+5. Please note that you will need to add the HAB_LICENSE environment variable to the generated chart (most likely in the templates/habitat.yaml file).
```
Comment (kagarmoe) [inline www/source/partials/docs/_dev-pkgs-exports.html.md.erb]:
```suggestion
5. Add the HAB_LICENSE environment variable to the generated chart, usually in the `templates/habitat.yml` file.
```

**Comment by nellshamrell** [comment]
Implemented @kagarmoe's suggestions!

**Pull request merged**